brooklyn-dist/vagrant/src/main/vagrant/servers.yaml

```diff
@@ -0,0 +1,47 @@
+---
+default_config:
+    check_newer_vagrant_box: true
+    run_os_update: true
+servers:
+  - name: brooklyn
+    box: ubuntu/vivid64
+    ram: 2048
+    cpus: 4
+    ip: 10.10.10.100
+    shell:
+      env:
+        BROOKLYN_VERSION: 0.9.0-SNAPSHOT
+      cmd:
+        - sudo sh -c 'export DEBIAN_FRONTEND=noninteractive; apt-get install --yes openjdk-8-jre-headless'
+        - curl -s -S -J -O -L "https://www.apache.org/dyn/closer.cgi?action=download&filename=brooklyn/apache-brooklyn-${BROOKLYN_VERSION}/apache-brooklyn-${BROOKLYN_VERSION}-bin.tar.gz"
```

Comment:
mmm -- don't much like that.  what about something like this:

    if [ ! -z "${BROOKLYN_VERSION##*-SNAPSHOT}" ] ; then 
        # url for official release versions
        BROOKLYN_URL=https://https://repository.apache.org/content/repositories/releases/org/apache/brooklyn/brooklyn-dist/${BROOKLYN_VERSION}/brooklyn-dist-${BROOKLYN_VERSION}-dist.tar.gz
    else
        # url for community-managed snapshots
        BROOKLYN_URL='https://repository.apache.org/service/local/artifact/maven/redirect?r=snapshots&g=org.apache.brooklyn&a=brooklyn-dist&v='${BROOKLYN_VERSION}'&c=dist&e=tar.gz'
    fi
    # alternatively if you want to use your local build, uncomment this line:
    # BROOKLYN_URL=~/.m2/repository/org/apache/brooklyn/brooklyn-dist/${BROOKLYN_VERSION}/brooklyn-dist-${BROOKLYN_VERSION}-dist.tar.gz

reminder you'll need to update the `curl` so it does a `-o brooklyn-dist-${BROOKLYN_VERSION}-dist.tar.gz` instead of `-O` for the latter command, as well as reading from "${BROOKLYN_URL}" of course.  (looks like it already follows links and ignores bad certs which is good.)